website/server/libs/email.js

```diff
@@ -47,8 +48,12 @@ export function getUserInfo (user, fields = []) {
   if (fields.indexOf('email') !== -1) {
     if (user.auth.local && user.auth.local.email) {
       info.email = user.auth.local.email;
-    } else if (user.auth.facebook && user.auth.facebook.emails && user.auth.facebook.emails[0] && user.auth.facebook.emails[0].value) {
-      info.email = user.auth.facebook.emails[0].value;
+    } else {
+      common.constants.SUPPORTED_SOCIAL_NETWORKS.forEach(network => {
+        if (user.auth[network.id] && user.auth[network.id].emails && user.auth[network.id].emails[0] && user.auth[network.id].emails[0].value) {
+          info.email = user.auth.facebook.emails[0].value;
```

Comment:
Is this right? shouldn't it be `user.auth[network.id].emails[0].value;`

Also, would it make sense to clean up this logic by using `_.get` here?